src/types/cart.d.ts

```diff
@@ -134,6 +134,10 @@ export interface CartEndpoint {
 
   UpdateItem(itemId: string, quantity: number, data?: any): Promise<CartItemsResponse>;
 
+  CreateCart(data: any, token?: string): Promise<CartItemsResponse>;
```

Comment:
Is the data really "any" here, or is it an array of products? I see we use any for other items so just confirming.